molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/admin/versions.html

```diff
@@ -0,0 +1,42 @@
+{% raw %}
+{% extends "base.html" %}
+{% load i18n %}
+{% block content %}
+<style>
+  table, th, td {
+      border: 1px solid black;
+  }
+  th, td {
+      padding: 5px;
+      font-size: 15px;
+  }
+  th {
+      text-align: left;
+  }
+</style>
```

Comment:
Note to myself: 
Remove the `style` tag and use something like the below implementation:
```
{% block extra_css %}
    {% include "wagtailadmin/pages/_editor_css.html" %}
{% endblock %}
```